package.json

```diff
@@ -21,7 +21,7 @@
     "compile:clean": "tsc --build tsconfig.json --clean",
     "watch": "tsc --build tsconfig.json --watch",
     "test": "jest",
-    "test:ci": "npm run test -- --ci --maxWorkers=2",
+    "test:ci": "npm run test -- --ci --runInBand",
```

Comment:
What's the reason we need to enable `--runInBand`? That can make running the tests slow, and more importantly is usually a sign tests aren't properly isolated.

Reply:
@martijnwalraven I should've made more clear in the PR / commit messages that this is just exploratory. After initial work, tests were passing in circle and failing in azure, so I was trying a few things to try and identify the issue. I don't intend for any testing config changes to be final for this PR.